src/smart-components/portfolio/remove-portfolio-modal.js

```diff
@@ -48,7 +49,21 @@ const RemovePortfolioModal = ({ viewState }) => {
     <UnauthorizedRedirect />
   ) : (
     <Modal
-      title={formatMessage(portfolioMessages.portfolioRemoveTitle)}
+      aria-label="Delete portfolio modal"
+      header={
+        <TextContent>
+          <Split hasGutter>
+            <SplitItem>
+              <ExclamationTriangleIcon size="lg" fill="#f0ab00" />
+            </SplitItem>
+            <SplitItem>
+              <Text component="h1" size="2xl">
+                {formatMessage(portfolioMessages.portfolioRemoveTitle)}
+              </Text>
+            </SplitItem>
+          </Split>
+        </TextContent>
```

Comment:
This markup seems too complicated.

You can check how it's done in Sources: https://github.com/RedHatInsights/sources-ui/blob/master/src/components/AddApplication/RemoveAppModal.js#L53

Reply:
Its PF layout. I think I will stick with it.